src/core/core.js

```diff
@@ -315,7 +315,7 @@ class Core {
       .reduce((chain, next) => chain.then(next), Promise.resolve())
       .catch(error => {
         // used for killing the run, no actual errors should be escalated here
-        log.debug(`run killed with: ${JSON.stringify(error)}`);
+        log.debug("run killed with: " + error);
```

Comment:
Eh, looks like Winston's default error object handling is just too simple. For an error that apparently kills the run, everything including the stacktrace should be printed and `JSON.stringify` won't work since the properties are not enumerable.

This looks like the simplest solution: https://github.com/winstonjs/winston/issues/1338#issuecomment-697958085
Would require also changing this to two log calls (also this feels more like error than debug level imo).

Reply:
Actually it does not kill the run, so the error message itself is wrong.

Reply:
Maybe it's supposed to? :D anyway, makes sense to just make it usable for now and reduce possible regressions

Reply:
I agree :)

Created a new issue from your comment to track it in future.